### DISCUSSION TITLE
Security Onion Integration

### DISCUSSION THREAD

**Discussion by Tranksley** [Security-Onion-Solutions/securityonion#9218] [Unsupported Versions]
Hello,
Being new to using security onion, I deployed it to a client to audit the network.

We are now in the integration phase.
 Indeed, we would like to integrate the network equipment and systems of this company with security Onion. 
Can you help me please with the steps to do this action?

**Comment by dougburks** [comment]
I'd start with our documentation:
https://docs.securityonion.net/en/2.3/getting-started.html
https://docs.securityonion.net/en/2.3/network.html
https://docs.securityonion.net/en/2.3/host.html
